react-web-app/package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "react-web-app-trustly-sample",
   "version": "0.1.0",
+  "engines": {
+    "node": ">=20"
```

Comment:
can we not support a lower version like 18 or 16 even? No big deal if not, but lower versions are still very common so it'd be nice not to force people to update if it's not necessary

Reply:
Node 16 should compile the project fine today, we don't have any breaking features so far.

My idea was to look forward by suggesting v20, but we can set the v16 as standard, no problem.

Reply:
Node version suggestion updated to the minimum version required ✅